### ISSUE TITLE
Enabling GTD terms does not apply to filters

### ISSUE THREAD

**Issue by ajfriesen** [mpcjanssen/simpletask-android#947]
When using the classic GTD terms you still see 'tags' and 'lists' in the filter action.

At least in the German translation.

Use the 'context' and 'project' in filters when the option for classic GTD terms is enabled.

**Comment by mpcjanssen** [comment]
What do you mean with filter action?

**Comment by ajfriesen** [comment]
Sorry, that was unclear.

Here is an image:

 ![lists and tags](https://i.imgur.com/jXUmmgX.jpg)

It shows:

- Zeige Listen der Aufgaben
- Zeige Tags der Aufgaben

**Comment by ddisanti** [comment]
> Maybe this is only wrong in the German translation

I don't use this feature but it happens in the Spanish translation too.

**Comment by mpcjanssen** [comment]
Yes there is no specific todo.txt version for those strings yet.

**Comment by emarsk** [comment]
> Yes there is no specific todo.txt version for those strings yet.

Actually, the [official specs page](https://github.com/todotxt/todo.txt) uses "project" and "context", so there should be at least an option to enable the official terminology.

It's a small thing, of course, but I'm also confused by "list" and "tag". I can somewhat see the argument for "tag" instead of "project", but to me "list" feels just wrong: it's completely generic and meaningless (isn't the entire todo.txt a list itself?) and the "list" is not even actually a real list, but a tag…, while "context" brings a very specific meaning: what I'm able to do in that place and/or that situation: a big point of GTD is focusing on what I'm able to do here and now.

**Comment by mpcjanssen** [comment]
@emarsk, there is a setting to use the official terminology. This issue describes a place where the setting has no effect yet.

**Comment by emarsk** [comment]
@mpcjanssen, I was commenting about the claim that "there is no specific todo.txt version for those strings". I interpreted this sentence as "there's no official terminology". Sorry if I misunderstood that.

**Comment by mpcjanssen** [comment]
@emarks no problem, just wanted to clear up any misunderstanding.

**Comment by emarsk** [comment]
By the way, the same non-official terminology is also present in the lua function names, at least according to the wiki. It would be nice to have aliases with the official names.

It would be extra nice - in my opinion - if the official terminology were the default one everywhere, because if you're basing a tool on a standard, it makes sense to have the defaults adhering to that standard as much as possible, and make your variations available as an option; if anything not to confuse newcomers.

**Comment by emarks** [comment]
> @emarks no problem, just wanted to clear up any misunderstanding.

Check username spelling on your reply.   emarks != emarsk

**Comment by mpcjanssen** [comment]
@emarsk making the default todo.txt terms available everywhere makes sense and I am all for it. I could even add a question on first launch to determine if the user wants todo.txt terms or generic terms.
I am not quite sure about doing this in code, because as it is the Lua parts reflect the actual naming in the Simpletask source code itself. It's should be fairly easy to add yourself though.

For the full rationale of the nameswitch see https://github.com/mpcjanssen/simpletask-android/blob/master/app/src/main/assets/listsandtags.en.md and keeping in mind that Simpletask is supposed to be a more generic tool than just using it for todo.txt or GTD.
